src/main/java/org/sagebionetworks/web/client/DisplayUtils.java

```diff
@@ -622,6 +622,7 @@ public static Tooltip addTooltip(Widget widget, String tooltipText, Placement po
 		t.setIsHtml(true);
 		t.setIsAnimated(false);
 		t.setTrigger(Trigger.HOVER);
+		t.setContainer("body");
```

Comment:
This is some Bootstrap magic, right?

Reply:
Yes

Reply:
I think the default behavior is to attach the new (absolute positioned) tooltip element as a sibling to the target element, but this allows you to specify another element.